src/components/SearchNftsTableCondensed.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <table
+      v-for="nft in nfts.data"
+      :key="nft.contractId"
+      class="search-nfts-table-condensed__table">
+      <tbody>
+        <tr class="search-nfts-table-condensed__row">
+          <th class="search-nfts-table-condensed__header">
+            <app-tooltip>
+              Symbol
+              <template #tooltip>
+                {{ nftsHints.symbol }}
+              </template>
+            </app-tooltip>
+          </th>
+          <td class="search-nfts-table-condensed__data">
+            <app-link
+              :to="`/nfts/${nft.contractId}`"
+              class="search-nfts-table-condensed__link">
+              {{ nft.symbol }}
+            </app-link>
+          </td>
+        </tr>
```

Comment:
```suggestion
```

I would remove the "Symbol" column to keep it consistent with the NFTs page where we don't have it.